### PR TITLE
chore: translations for login.properties

### PR DESCRIPTION
**Pull request by ui-okta**
## Description
Auto copy of latest translations by okta-i18n script.
### Bacon task details
- Task ID: 571bf846a56240ea84f7eecb433e4eb3 
- Requested by: evelyn.teo@okta.com

## Notification sent to
@okta/nxt @haishengwu-okta @nikhilvenkatraman-okta @aarongranick-okta @mauriciocastillosilva-okta

### CODE REVIEW

File: packages/@okta/i18n/src/properties/login_cs.properties [hunk 1]
```diff
@@ -875,6 +875,7 @@ common.logo.alt = logo aria
 ## Generic
 oie.authenticator.enroll.error.fail = Ov\u011B\u0159ovac\u00ED aplikaci nelze zaregistrovat. Zkuste to znovu.
 oie.authenticator.verify.error.fail = Ov\u011B\u0159ovac\u00ED aplikaci nelze ov\u011B\u0159it. Zkuste to znovu.
+oie.configuration.error = N\u011Bco se nepovedlo. Byla zji\u0161t\u011Bna potenci\u00E1ln\u011B chybn\u00E1 konfigurace. Kontaktujte pros\u00EDm podporu.
```
Comment (lokeshhemrajani-okta) [inline packages/@okta/i18n/src/properties/login_cs.properties:879]: This is something which I did not change in login.properties. Do we still need to take this or any other team requested?

**Review by lokeshhemrajani-okta — COMMENTED** [review]
There are conversions which are not requested for my story but I still see those. Do we have to take those?

File: packages/@okta/i18n/src/properties/login_cs.properties (hunk 1 above)
Comment (lokeshhemrajani-okta) [reply packages/@okta/i18n/src/properties/login_cs.properties:879]: @lesterchoi-okta

Comment (evelynteo-okta) [reply packages/@okta/i18n/src/properties/login_cs.properties:879]: @lokeshhemrajani-okta  Please include in your PR. There is no 1:1 PR content mapping. Language quality improvement  is an ongoing effort.

**Review by lokeshhemrajani-okta — APPROVED** [review]

**Review by lesterchoi-okta — APPROVED** [review]

**Pull request closed without merging**